ai-agents/fhir-questionnaire-generation/data_ingestion_pipeline/pdf_to_md_service/utils.py

```
@@ -40,3 +42,63 @@ def convert_pdf_to_markdown(pdfFile) -> str:
+                await send_notification(job_id, file_name, "completed", "pdf_to_md_done", configs, logger)
+                if configs.USE_FTP:
+                    await delete_pdf_file_ftp(file_name, configs, logger)
+                else:
+                    await delete_pdf_file_local(file_name, configs, logger)
+            else:
+                logger.error(f"Job {job_id} failed during storage")
+                await send_notification(job_id, file_name, "failed", "storage_failed", configs, logger)
+        else:
+            logger.error(f"Job {job_id} failed during conversion")
+            await send_notification(job_id, file_name, "failed", "conversion_failed", configs, logger)
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)